src/movies.js

```diff
@@ -1,25 +1,175 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
 
+function getAllDirectors(moviesArray) {
+    let allDirectors=moviesArray.map(x=>x.director)
```

Comment:
Porque let!!! si no se reasigna el valor siempre siempre siempre usa const!

```suggestion
    const allDirectors=moviesArray.map(x=>x.director)
```